index.html

```diff
@@ -196,6 +210,8 @@ <h2 class="title-about">About<br>Myself</h2>
         </li>
        </ul>
        <div><button class="sec-button">Get my resume</button></div>
+      </div>
+       <div class="new-lang">
        <div class="accordion"><p>Language</p><svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" stroke-width="1.5" stroke="currentColor" class="icon">
         <path stroke-linecap="round" stroke-linejoin="round" d="M19.5 8.25l-7.5 7.5-7.5-7.5" />
```

Comment:
Please make sure to follow the exact font size in your language icons as well.

![image](https://user-images.githubusercontent.com/34779928/194394140-af495ff4-9142-42ad-b65f-76886b868564.png)